dashboard/app/assets/stylesheets/application.scss

```diff
@@ -1730,7 +1729,31 @@ a.download-video {
   .devise_links {
     padding-left: 20px;
     margin-top: 20px;
-    border-left: 1px solid $black;
+  }
+}
+
+.devise_link_section {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
```

Comment:
Not sure if I mentioned before, but some of the earliest browsers we support have some limitations on their flexbox support and we should test explicitly.  I think @islemaster knows the most about this.

Reply:
I've found http://caniuse.com/#feat=flexbox to be fairly accurate.  Only IE11 is likely to cause issues for us, only in the following scenarios:

- IE 11 requires a unit to be added to the third argument, the flex-basis property
- IE 11 does not vertically align items correctly when min-height is used
- In IE10 and IE11, containers with display: flex and flex-direction: column will not properly calculate their flexed childrens' sizes if the container has min-height but no explicit height property.

So generally `flex-direction: column` and `min-height` together is a red flag.  I don't see that here, so this is likely to work fine.  Worth testing though :+1:.